src/Diagnostics.RuntimeHost/appsettings.json

```diff
@@ -104,5 +104,9 @@
     "ClientId": "",
     "AllowedAppIds": ""
   },
-  "CloudDomain": "PublicAzure"
+  "CloudDomain": "PublicAzure",
+  "HealthCheckSettings": {
+    "IsOutboundConnectivityCheckEnabled": true,
+    "OutboundConnectivityCheckUrl": "http://google.com/generate_204"
```

Comment:
Just curious, how did you hear about this endpoint?

Reply:
https://stackoverflow.com/questions/2031824/what-is-the-best-way-to-check-for-internet-connectivity-using-net